<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>12</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Barcoder</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>533</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPushButton" name="btn_file_attach">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Liberation Sans</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Подключить Excel-файл</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="lb_filename">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Liberation Mono</family>
          <pointsize>8</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_file_preview">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Liberation Sans</family>
          <pointsize>12</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>SizeFDiagCursor</cursorShape>
        </property>
        <property name="text">
         <string>Просмотр</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="gbox_type">
      <property name="font">
       <font>
        <family>Liberation Sans</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Тип этикетки</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QComboBox" name="cmb_type">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>50</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbox_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Liberation Sans</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Размер этикетки</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="cmb_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>50</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>800</width>
           <height>600</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbox_qty_mode">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Liberation Sans</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Кол-во этикеток</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="cmb_qty_mode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>50</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>800</width>
           <height>600</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_create">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>40</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>65</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Liberation Sans</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Создать файл со Штрихкодами</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="font">
     <font>
      <family>Liberation Sans</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="menu_file_attach"/>
    <addaction name="menu_file_unattach"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="font">
     <font>
      <family>Liberation Sans</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="menu_help_tutor"/>
    <addaction name="menu_help_about"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Liberation Sans</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <action name="menu_file_attach">
   <property name="text">
    <string>Подключить</string>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </action>
  <action name="menu_file_unattach">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </action>
  <action name="menu_help_tutor">
   <property name="text">
    <string>Инструкция</string>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </action>
  <action name="menu_help_about">
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
